hello.cpp

```diff
@@ -1,28 +1,46 @@
+#include <cstring>
 #include <launchdarkly/api.hpp>
 
-const char *username = "jeff@example.com";
+// Set MOBILE_KEY to your LaunchDarkly mobile key.
+#define MOBILE_KEY "YOUR_MOBILE_KEY"
 
-// update me to your own keys
-const char *sdk_key = "YOUR_SDK_KEY";
-const char *feature_key = "YOUR_FEATURE_KEY";
+// Set FEATURE_KEY to the feature flag you want to evaluate.
+#define FEATURE_KEY "YOUR_FEATURE_KEY"
 
 int main() {
+
+    if (strlen(MOBILE_KEY) == 0) {
```

Comment:
Sorry just noticed this -- I ran this locally on re-review, and this check doesn't have the desired behavior now that `MOBILE_KEY` starts out as `"YOUR_MOBILE_KEY"`. Can you set line 5 to empty string?